core/src/test/resources/ExplicitImplicit/basic.source

```diff
@@ -44,14 +71,124 @@ class A {
 class A {
   class B { class C }
   implicit val x = new B
-  implicit val y = new x.C
+  implicit val y: x.C = new x.C
 }
 >>>
 class A {
   class B { class C }
   implicit val x: B = new B
   implicit val y: x.C = new x.C
 }
+<<< NOWRAP higher kinded cats
+package cats {
```

Comment:
This is just a reproduction of an intricate scenario. Even though it has the names cats, I would leave it here. If you disagree, please point to concrete file you'd like to see have it.